src/k8s-extension/azext_k8s_extension/partner_extensions/DataProtectionKubernetes.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-argument
+from knack.log import get_logger
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.azclierror import RequiredArgumentMissingError, InvalidArgumentValueError
+
+from .DefaultExtension import DefaultExtension
+from .._client_factory import cf_storage, cf_managed_clusters
+from ..vendored_sdks.models import (Extension, PatchExtension, Scope, ScopeCluster)
+
+logger = get_logger(__name__)
+
+
+class DataProtectionKubernetes(DefaultExtension):
+    def __init__(self):
+        """Constants for configuration settings
+           - Tenant Id (required)
+           - Backup storage location (required)
+           - Resource Limits (optional)
+        """
+        self.TENANT_ID = "credentials.tenantId"
+        self.BACKUP_STORAGE_ACCOUNT_CONTAINER = "configuration.backupStorageLocation.bucket"
+        self.BACKUP_STORAGE_ACCOUNT_NAME = "configuration.backupStorageLocation.config.storageAccount"
+        self.BACKUP_STORAGE_ACCOUNT_RESOURCE_GROUP = "configuration.backupStorageLocation.config.resourceGroup"
+        self.BACKUP_STORAGE_ACCOUNT_SUBSCRIPTION = "configuration.backupStorageLocation.config.subscriptionId"
+        self.RESOURCE_LIMIT_CPU = "resources.limits.cpu"
+        self.RESOURCE_LIMIT_MEMORY = "resources.limits.memory"
+
+        self.blob_container = "blobContainer"
+        self.storage_account = "storageAccount"
+        self.storage_account_resource_group = "storageAccountResourceGroup"
+        self.storage_account_subsciption = "storageAccountSubscriptionId"
+        self.cpu_limit = "cpu"
```

Comment:
rename to cpuLimit
and memoryLimit resp?

Reply:
Done